components/ui/accordion/accordion.tsx

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
Oooooo I see that u built the accordion yourself. I think going forward we should use the https://ui.shadcn.com/ library as much as possible. it gives us prebuilt components that can be fully customized but come with inbuilt and consistent style out of the box that I really like overall. It also saves us a lot of time (most of the time)

Most components in the components/ui folder are shadcn components. If we build reusable components from scratch we have been putting them in the components/custom folder. Sorry if this was not more clear ahaha

Reply:
Hahahaha all good man. I've switched over to shadcn now and when I've got a spare moment I can move my custom components from ui to custom :)

Reply:
This makes me think that we should rename ui directory to something like shadcn that is more clear about what should go there cause I understand why ui and custom does not have distinct boundaries

You can also customize shadcn components (as I did in my header PR) if we need to